modules/serverless_negs/README.md

```diff
@@ -17,7 +17,7 @@ need a Terraform 0.11.x-compatible version of this module, the last released ver
 ```HCL
 module "lb-http" {
   source            = "GoogleCloudPlatform/lb-http/google//modules/serverless_negs"
-  version           = "~> 3.1"
+  version           = "~> 4.4"
```

Comment:
@bharathkkb I think this still will won't work for `serverless_negs` submodule because it's not in a tagged release yet?

Reply:
@ahmetb thanks for catching this. We should update once we cut the 4.5. Once we have 4.5, `~> 4.4` should work, but we should still update it for clarity.

Reply:
We should update the release-please automation to automatically update the submodule versions as well: https://github.com/googleapis/release-please/issues/637